jit-persistence/src/main/java/org/dcsa/jit/persistence/repository/TimestampInfoRepository.java

```diff
@@ -0,0 +1,23 @@
+package org.dcsa.jit.persistence.repository;
+
+import org.dcsa.jit.persistence.entity.TimestampInfo;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.data.jpa.repository.EntityGraph;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface TimestampInfoRepository
+    extends JpaRepository<TimestampInfo, UUID>,
+            JpaSpecificationExecutor<TimestampInfo> {
+
+  @EntityGraph(value = "graph.allAttributes")
+  List<TimestampInfo> findAll(Sort sort);
+
+  @EntityGraph(value = "graph.allAttributes")
+  List<TimestampInfo> findAll(Specification<TimestampInfo> spec, Sort sort);
```

Comment:
This can be removed.

Reply:
No, it is used by UI-Support (via its timestamp-info endpoint)